client/src/pages/Review/Review.tsx

```diff
@@ -221,16 +222,18 @@ const Review = () => {
             </main>
           </motion.main>
         ) : (
-          <div
-            className="max-w-9xl mx-auto"
-            ref={mainRef}
-            style={{ height: "100vh" }}
-          >
-            <Dropzone
-              onDrop={onOutlineUpload}
-              isLoading={loading.length > 0}
-            />
-          </div>
+          !isMobile() && (
```

Comment:
No need to render the empty state on mobile, since it's behind the NavigationPanel anyways